packages/crypto-react-native/src/utils.ts

```diff
@@ -67,3 +67,15 @@ export const bs58ToBin = (bs58Address: string): Buffer => {
 
   return payload
 }
+
+export const bs58KeyType = (bs58Address: string): number => {
+  const bin = bs58ToBin(bs58Address)
```

Comment:
Do these throw if the  bs58Address string is invalid?

Reply:
Yes, the bs58ToBin function throws in 2 places, 1) if it cannot decode bs58 and 2) if the checksum is invalid